pkgs/os-specific/linux/lenovo-legion/app.nix

```
@@ -0,0 +1,40 @@
+{ lib, fetchurl, xorg, libsForQt5, wrapQtAppsHook, python3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "lenovo-legion-app";
+  version = "main2023-04-02-16-53-51";
```

```suggestion
  version = "2023-04-02-16-53-51";
```
I would short the release version to that intuitively but I didn't look at the repo so that change might be bad.

---

This is the name of the tag, but removing `main` seems reasonable.